layers/lcm.go

```diff
@@ -191,3 +206,8 @@ func (lcm LCM) LayerPayload() []byte {
 func (lcm LCM) Payload() []byte {
 	return lcm.LayerPayload()
 }
+
+// Fingerprint returns the LCM fingerprint of the underlying message.
+func (lcm LCM) Fingerprint() LCMFingerprint {
+	return lcm.fingerprint
```

Comment:
just export (capitalize) this value?  that way folks can set it as well as get it, which would be nice if we ever make this serializable.

Reply:
@gconnell thanks for the response! :sunglasses: 
That would be a larger change, and have effect on the `lcmLayerTypes` map. I agree that serializeable is nice, but that is IMHO out of scope for this change. 
I'd rather just have it as is in this PR, and if / when its made serializeable we can reevaluate.